maglink.py

```diff
@@ -26,19 +26,17 @@ def directorLink(magiclinks, numLinks, event):
         permissions = []
         for i in event['permissions']:
              permissions.append(i)
-        for j in range(numLinks):
+        for j in range(min(numLinks, len(event['emailsTo']))):
```

Comment:
do we need this min()

Reply:
It's to make sure we don't go out of bounds in the `emailsTo` list.

Reply:
I guess we _could_ get rid of `numLinks`?